site/_includes/footer.html

```diff
@@ -33,7 +33,7 @@
   <div class="footer-row" id="footer_bottom">
     <div class="footer-legal">
       <span class="copyright">
-        &copy; 2016 ManageIQ.
+        &copy; 2017 ManageIQ.
```

Comment:
or perhaps `Date.current.year` ? we don't need the hours/minutes details and it takes time zones into consideration